src/app/cards/[card_id]/_components/CardPage.tsx

```diff
@@ -76,7 +86,12 @@ export const CardPage = () => {
         >
           拡大表示
         </Button>
-        <Button leftSection={<IconBallpen />} fullWidth variant="outline">
+        <Button
+          leftSection={<IconBallpen />}
+          fullWidth
+          variant="outline"
+          onClick={handleOnClickEdit}
```

Comment:
```suggestion
component={Link}
href={ROUTES.cards.edit(params.card_id)}
```
router.pushを使わなくていいなら基本使わないようにおなしゃす

Reply:
それに応じて`handleOnClickEdit`も削除